src/geophires_x/SUTRAOutputs.py

```diff
@@ -156,7 +156,7 @@ def PrintOutputs(self, model: Model):
                 f.write(f"      Well Depth:                                    {model.reserv.depth.value:10.1f} " + model.reserv.depth.CurrentUnits.value + NL)
 
                 pump_efficiency_display_unit = model.surfaceplant.pump_efficiency.CurrentUnits.value
-                pump_efficiency_display = f'{model.surfaceplant.pump_efficiency.value:10.1f} {pump_efficiency_display_unit}'
+                pump_efficiency_display = f'{model.surfaceplant.pump_efficiency.value * 100:10.1f} {pump_efficiency_display_unit}'
```

Comment:
In theory we shouldn't have to multiply by 100 here because conversion from current units to preferred units should be happening beforehand and doing it for us. Do you know why that apparently isn't the case?

Reply:
I tried this, but it does not produce the desired output.  Without the explicit multiplication by 100 (as in line 159 above), pump efficiency is output as `0.8 %`.

Reply:
Sounds like it'll require debugger analysis